packages/adp-tooling/src/preview/adp-preview.ts

```diff
@@ -75,7 +76,18 @@ export class AdpPreview {
         private readonly util: MiddlewareUtils,
         private readonly logger: ToolsLogger
     ) {
+        dotenv.config();
         this.routesHandler = new RoutesHandler(project, util, logger);
+
+        if (!config.auth && process.env.FIORI_TOOLS_USER && process.env.FIORI_TOOLS_PASSWORD) {
+            config.auth = {
+                username: process.env.FIORI_TOOLS_USER,
+                password: process.env.FIORI_TOOLS_PASSWORD
+            };
+        } else if (config.auth && !process.env.FIORI_TOOLS_USER && !process.env.FIORI_TOOLS_PASSWORD) {
```

Comment:
This can be removed if we don't support providing the variables as part of the yaml.
Btw, if the user/password are missing, then it will already be prompted and stored in the variables (https://github.com/SAP/open-ux-tools/blob/main/packages/system-access/src/base/connect.ts#L105-L108)